studybuddies/src/app/app.module.ts

```diff
@@ -11,10 +13,12 @@ import { CreateClassStudentComponent } from './components/student/create-class-s
 import { ClassListTutorComponent } from './components/tutor/class-list-tutor/class-list-tutor.component';
 import { CreateClassTutorComponent } from './components/tutor/create-class-tutor/create-class-tutor.component';
 import { HomeComponent } from './components/home/home.component';
+import { LoginComponent } from './components/login/login.component';
 
 const routes: Routes = [
   { path: 'student', component: StudentComponent },
   { path: 'tutor', component: TutorComponent },
+  { path: 'login', component: LoginComponent },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167323)
```suggestion
  { path: "login", component: LoginComponent },
```